### DISCUSSION TITLE
Gmail: self signed certificate error

### DISCUSSION THREAD

**Discussion by jmperou** [jef/streetmerchant#1186] [Help]
This may be a configuration issue on my end but unable to find solution for it. Would anyone be able to confirm how to resolve my gmail certificate issue? 

[8:10:31 PM] info :: 🚀🚨 [amazon] [pny (3070)] xlr8 revel :: IN STOCK 🚨🚀
https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08HBJB7YD&Quantity.1=1
[8:10:31 PM] info :: ✔ desktop notification sent
[8:10:31 PM] error :: ✖ couldn't send email self signed certificate in certificate chain {
  "code": "ESOCKET",
  "command": "CONN",
  "stack": "Error: self signed certificate in certificate chain\n    at TLSSocket.onConnectSecure (_tls_wrap.js:1497:34)\n    at TLSSocket.emit (events.js:315:20)\n    at TLSSocket._finishInit (_tls_wrap.js:932:8)\n    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706:12)"

**Comment by amador-e-rivera** [comment]
Not sure if OP solved this problem but if someone comes across this issue, you can try disabling your anti-virus program.
